.github/workflows/publish.yml

```
@@ -0,0 +1,149 @@
+            echo "Run on schedule; going to skip this run, as this is a fork"
+            SKIP="true"
+          else
+            SKIP="false"
+          fi
+        else
+          FOLDER="${REPO_NAME}-releases"
+          SKIP="false"
+        fi
+
+        echo "::set-output name=release-date::${RELEASE_DATE}"
+        echo "::set-output name=version::${VERSION}"
+        echo "::set-output name=folder::${FOLDER}"
+        echo "::set-output name=dry-run::${DRY_RUN}"
+        echo "::set-output name=skip::${SKIP}"
+
+        echo "Release-date: ${RELEASE_DATE}"
+        echo "Version: ${VERSION}"
+        echo "Folder: ${FOLDER}"
+        echo "Dry-run: ${DRY_RUN}"
+        echo "Skip: ${SKIP}"
+
+    - if: steps.vars.outputs.skip == 'false'
+      name: Install dependencies
+      run: |
+        set -e
+
+        sudo apt update
+        sudo apt install -y catcodec --no-install-recommends
+
+    - if: steps.vars.outputs.skip == 'false'
+      name: Build
+      run: |
+        set -e
+
+        make maintainer-clean
+        make bundle_zip bundle_xsrc
+
+        # Move bundles in their own folder
+        mkdir bundles
+        mv opensfx-${{ steps.vars.outputs.version }}-source.tar.xz bundles/
+        mv opensfx-${{ steps.vars.outputs.version }}.zip bundles/opensfx-${{ steps.vars.outputs.version }}-all.zip
+
+    - if: steps.vars.outputs.skip == 'false'
+      name: Create checksums
+      run: |
+        set -e
+
+        cd bundles
+        for i in $(ls); do
+          openssl dgst -r -md5 -hex $i > $i.md5sum
+          openssl dgst -r -sha1 -hex $i > $i.sha1sum
+          openssl dgst -r -sha256 -hex $i > $i.sha256sum
+        done
+
+        # Show the content of the bundles folder, to make problems debugging easier
+        ls -l
+
+    - if: steps.vars.outputs.skip == 'false'
+      name: Prepare bundles folder
+      run: |
+        set -e
+
+        echo "${{ steps.vars.outputs.release-date }}" > bundles/released.txt
+        cp docs/readme.txt bundles/
+        cp docs/changelog.txt bundles/
+
+        # Show the content of the bundles folder, to make problems debugging easier
+        ls -l bundles/
+
+    - if: steps.vars.outputs.dry-run == 'false' && steps.vars.outputs.skip == 'false'
+      name: Publish
+      id: publish
+      run: |
+        set -e
+
+        aws s3 cp --recursive --only-show-errors bundles/ s3://${{ secrets.CDN_S3_BUCKET }}/${{ steps.vars.outputs.folder }}/${{ steps.vars.outputs.version }}/
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    - if: steps.vars.outputs.dry-run == 'false' && steps.vars.outputs.skip == 'false'
+      name: Trigger 'update CDN'
+      uses: peter-evans/repository-dispatch@v1
+      with:
+        token: ${{ secrets.DEPLOYMENT_TOKEN }}
+        repository: OpenTTD/workflows
+        event-type: update_cdn
```

Issue spotted in OpenGFX, and as I copied the workflow :)
```suggestion
        event-type: update-cdn
```